### PR TITLE
graphviz: update to 10.0.1

### PR DESCRIPTION
**Pull request by Suyun114**
Topic Description
-----------------

- graphviz: update to 10.0.1

Package(s) Affected
-------------------

- graphviz: 10.0.1

Security Update?
----------------

No

Build Order
-----------

```
#buildit graphviz
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`

### CODE REVIEW

File: app-creativity/krita/autobuild/defines [hunk 1]
```diff
@@ -5,7 +5,7 @@ PKGDEP="boost exiv2 fftw giflib gsl kcompletion kconfig kcoreaddons kcrash \
         kwindowsystem lcms2 libheif libjpeg-turbo libjxl libmypaint libraw \
```
Comment (MingcongBai) [inline app-creativity/krita/autobuild/defines]: Why is this here?

**Review by MingcongBai — DISMISSED** [review]

**Review by jiegec — APPROVED** [review]
